rust/scripts/build-release.sh

```diff
@@ -64,6 +64,10 @@ main() {
         find . -type f -name "lib$1.a"
     fi
 
+    # generate filcrypto.h
+    RUSTFLAGS="${__rust_flags}" HEADER_DIR="." \
+        cargo test build_headers --features c-headers 
```

Comment:
@dignifiedquire why build with a test?

Reply:
Basically, it means we need to rebuild _everything_ (and we should just be using `cargo run` with a bin target).

Reply:
Mostly because that is how the `safer-ffi` docs do it, and it didn't quite work when doing it differently due to how that project currently works.